meetings/2024-01-10.md

```diff
@@ -75,33 +90,65 @@ URL: https://virginiabalseiro.github.io/solid-ecosystem-monitor/
 
 Proposed by: Fred Gibson
 
-* FG: Enabling information interoperability by thinking of a pod as the complete space-time model for a part of the world where the pod has two interconnecting hierarchies, a hierarchy of files/documents in ldp containers with root at /, and a hierarchy of system relations with root at the webid for the system that the pod represents.  A demo of this approach in action using a Car Dealership building on a TrinPod.
-
+* FG: FG: Enabling information interoperability by thinking of a pod as the complete space-time model for a part of the world where the pod has two interconnecting hierarchies: one hierarchy of files/documents in ldp containers with root at `/`; and a second hierarchy of system relations with root at the webid for the system that the pod represents.  A demo of this approach is in action using a Car Dealership building on a TrinPod.
+* MdJ: Am working on documenting using your ontology. Regarding private attributes and search, question for the group: we're sort of re-organised now, who would pick up on this - can we evolve solid to those things that are needed in practise?
+* eP: Great presentation. Nice UI. Important for UX, and nice demos to see in Solid. Seems you have specific knowledge domain, for a lot of people also doing clone of X clone, and Solid perhaps more complex, or the scope the broader than just social media. Great to see use cases like yours. Regarding sub-resources access control, would like to hear about your experiences on that. What kind of intearctions work for the user and how the document hierarchy interacts with the knowledge hierarchy?
+* FG: We have CSV upload. Simplified way to put triples into the system. A lot of problems to solve for this to work. One is ontologies. This creates extra bandwidth but we deliver an onotology where each time you make a request, it also delivers the (complete) ontology needed. We only create what doesn't exist. Our system ontology worked for that. We have conceptual "ids", things like "t_type" are just there as shortcut. This would be for a sophisticated user to set this up. You can of course do all this with RDF, this was a quick and easy UI. Also have commands in a row-based instructions in a spreadsheet. We'll increasingly add automation. The ontology that would allow interoperability.
```

Comment:
```suggestion
* FG: We have CSV upload. Simplified way to put triples into the system. A lot of problems to solve for this to work. One is ontologies. This creates extra bandwidth but we deliver an ontology where each time you make a request, it also delivers the (complete) ontology needed. We only create what doesn't exist. Our system ontology worked for that. We have conceptual `ids`, things like `t_type` are just there as shortcut. This would be for a sophisticated user to set this up. You can of course do all this with RDF; this was a quick and easy UI. Also have commands in row-based instructions in a spreadsheet. We'll increasingly add automation. The ontology would allow interoperability.
```